include/fsm/fsm.h

```diff
@@ -114,7 +114,7 @@ fsm_addstate_bulk(struct fsm *fsm, size_t n);
  /*
  * Remove a state. Any edges transitioning to this state are also removed.
  */
-void
+int
 fsm_removestate(struct fsm *fsm, fsm_state_t state);
```

Comment:
As discussed (on slack), the only current caller for `fsm_removestate` is `fsm_mergestates`, which does not appear to be used. We could potentially remove both from the public interface. `fsm_removestate` doesn't really fit well with the new continuous array of states implementation, we tend to compact on copy instead.